msgpack/_unpacker.pyx

```diff
@@ -314,6 +315,7 @@ cdef class Unpacker(object):
     cdef object object_hook, object_pairs_hook, list_hook, ext_hook
     cdef object encoding, unicode_errors
     cdef Py_ssize_t max_buffer_size
+    cdef uint64_t stream_offset
```

Comment:
`Py_ssize_t` may be 32bit, which is a bit small even today. 16 Exabyte should be plenty for some time.